terraform/variables.tf

```diff
@@ -40,11 +44,13 @@ variable "mysql_router" {
     base        = optional(string, "ubuntu@22.04")
     channel     = optional(string, "dpe/candidate")
     config      = optional(map(string), {})
-    constraints = optional(string, "arch=amd64")
```

Comment:
I do not recall detail, but AFAIR there were issues without it (maybe when arm revision release as bigger ID)... do not recall. Re-share it please if detect it again. :-D

Reply:
For the history: https://github.com/canonical/mysql-k8s-bundle/pull/122/files#r2422054030